recipes/perl-autovivification/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set perl_name = "autovivification" %}
+{% set name = "perl-" + perl_name %}
+{% set author = "VPIT" %}
+{% set version = "0.18" %}
+{% set sha256 = "2d99975685242980d0a9904f639144c059d6ece15899efde4acb742d3253f105" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/{{ author[0] }}/{{ author[0:2] }}/{{ author }}/{{ perl_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: |
+    perl Makefile.PL INSTALLDIRS=vendor
+    make
+    make test
+    make install
+  skip: true  # [win]
+  number: 0
+  run_exports:
+    weak:
+      - {{ name }} ={{ version }}
```

```suggestion
      - perl-autovivification ={{ version }}
```
See above comment about using less Jinja